### PR TITLE
chore: 🐛 fix pre-commit hook for file renames

### PR DESCRIPTION
**Pull request by ghusse**
Fixes the pre-commit hook that fails when there are renamed files to be committed.

## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: .eslint-bin/pre-commit-hook.js [hunk 1]
```diff
@@ -19,7 +19,17 @@ function getFilesModified(callback) {
...
+        if (file.index === 'R') {
+          // NOTICE: file path equals "old/path.js -> new/path.js" for renames
+          const matches = file.path.match(/^(?:.* -> )?(.*)$/);
+
+          return matches[1];
+        }
+
+        return file.path;
+      })
+      .filter(Boolean)
```
Comment (arnaudbesnier) [inline .eslint-bin/pre-commit-hook.js:32]: What is it used for?
Does it fix another use case?

I can't see this in our other repositories (https://github.com/ForestAdmin/forestadmin-server/commit/df1442a2ec1717c96be3b8e3bdcb11e1f2325029)

**Comment by ghusse** [comment]
> Just a warning about your PR comment that is not a "conventional commit" and could be rejected on merge (if not reworked before the GitHub "squash and merge" action)

Thanks @arnaudbesnier, I edited the name

File: .eslint-bin/pre-commit-hook.js (hunk 1 above)
Comment (ghusse) [reply .eslint-bin/pre-commit-hook.js:32]: I don't really think that it's usefull in real-life situations, but I wanted to be sure that if the regexp did not capture the file path, an `undefined` value is not added to the final array.

Do you prefer that I remove this line?

Comment (ghusse) [reply .eslint-bin/pre-commit-hook.js]: I answered my own question and removed the test.

**Pull request merged**